src/main/scala/org/labrad/manager/auth/AuthServer.scala

```diff
@@ -158,13 +183,22 @@ class AuthServer(
       auth.changePassword(username, oldPassword, newPassword)
     }
```

Comment:
Not part of this PR, but this looks like a bug.  oldpassword should not be required for admin users.